src/pages/restaurant/components/CartTotal/index.js

```diff
@@ -0,0 +1,68 @@
+import React, { useMemo } from 'react'
+import { useSelector } from 'react-redux'
+import PropTypes from 'prop-types'
+
+import { Products } from '../../reducers/selectors/products.selector'
+import { Cart } from '../../reducers/selectors/cart.selector'
+import FullWidthButton from '../buttons/FullWidthButton'
+import { DEFAULT_PROP } from '../../constants/globalConstants'
+
+import "./style.css"
+
+const getTotalAmount = (cartItem, productDetails) => cartItem.orderQuantity * parseInt(productDetails.price)
+
+const getCartTotalPrice = (cartList, productList) => cartList.reduce((total,cartItem) => total += getTotalAmount(cartItem, productList[cartItem.itemId]) ,0)
+
+const mapCartItems = (order,products)=> <AmountDetail key={order.itemId} itemName={products[order.itemId].name} amount={getTotalAmount(order, products[order.itemId])} />
+
+const AmountDetail = ({itemName, amount}) => {
+    return (
+        <div className="price-row-container">
+            <div className="price-row-detail">{itemName}</div>
+            <div className="price-row-amount">{amount}</div>
+        </div>
+    )
+}
+
+const CartTotal = (props) => {
+    
+    const products = useSelector(Products)
+    const cart = useSelector(Cart)
+
+    let totalAmount = useMemo(()=>getCartTotalPrice(cart,products),[cart,products])
+    const {handleOrderNowClick} = props
```

Comment:
handleOrderNowClick =>  onClick
**handle** we write wherever we handle the event, which is in parent component. but prop should be passed as  onClick